p2p/src/peer.rs

```diff
@@ -29,39 +32,38 @@ use tokio::{
 
 use crate::{
     network::{ConnectionId, PeerMessage, Post, Start, StopSelf},
-    Error, Message, MessageResult,
+    CryptographicError, Error, HandshakeError, Message, MessageResult,
 };
 
 /// Max message length in bytes.
-const MAX_MESSAGE_LENGTH: usize = 16 * 1024 * 1024;
-const MAX_HANDSHAKE_LENGTH: usize = 255;
+pub const MAX_MESSAGE_LENGTH: usize = 16 * 1024 * 1024;
+/// Max length of message in bytes.
+pub const MAX_HANDSHAKE_LENGTH: usize = 255;
```

Comment:
Shouldn't this say "length of handshake"?

Reply:
Not really. The handshake length could be shorter.